### PR TITLE
[Feature][FUN] add breedersInSpace splash message

### PR DESCRIPTION
**Pull request by flx-sta**
# Missing translations

- [x] Spanish
- [x] Italian
- [x] Korean
- [x] Chinese
- [ ] Chinese TW

---

## What are the changes?

added a `Breeders in space!` splash screen message

## Why am I doing these changes?

fun

## What did change?

see "What are the changes?"

### Screenshots/Videos

image
image
image

## How to test the changes?

replace `src/data/splash-messages.ts` with this:
```ts
import i18next from "i18next";

export function getBattleCountSplashMessage(): string {
  return `{COUNT} ${i18next.t("splashMessages:battlesWon")}`;
}

export function getSplashMessages(): string[] {
  const splashMessages = Array(10).fill(getBattleCountSplashMessage());
  splashMessages.push(...[
    i18next.t("splashMessages:joinTheDiscord"),
    i18next.t("splashMessages:infiniteLevels"),
    i18next.t("splashMessages:everythingStacks"),
    i18next.t("splashMessages:optionalSaveScumming"),
    i18next.t("splashMessages:biomes"),
    i18next.t("splashMessages:openSource"),
    i18next.t("splashMessages:playWithSpeed"),
    i18next.t("splashMessages:liveBugTesting"),
    i18next.t("splashMessages:heavyInfluence"),
    i18next.t("splashMessages:pokemonRiskAndPokemonRain"),
    i18next.t("splashMessages:nowWithMoreSalt"),
    i18next.t("splashMessages:infiniteFusionAtHome"),
    i18next.t("splashMessages:brokenEggMoves"),
    i18next.t("splashMessages:magnificent"),
    i18next.t("splashMessages:mubstitute"),
    i18next.t("splashMessages:thatsCrazy"),
    i18next.t("splashMessages:oranceJuice"),
    i18next.t("splashMessages:questionableBalancing"),
    i18next.t("splashMessages:coolShaders"),
    i18next.t("splashMessages:aiFree"),
    i18next.t("splashMessages:suddenDifficultySpikes"),
    i18next.t("splashMessages:basedOnAnUnfinishedFlashGame"),
    i18next.t("splashMessages:moreAddictiveThanIntended"),
    i18next.t("splashMessages:mostlyConsistentSeeds"),
    i18next.t("splashMessages:achievementPointsDontDoAnything"),
    i18next.t("splashMessages:youDoNotStartAtLevel"),
    i18next.t("splashMessages:dontTalkAboutTheManaphyEggIncident"),
    i18next.t("splashMessages:alsoTryPokengine"),
    i18next.t("splashMessages:alsoTryEmeraldRogue"),
    i18next.t("splashMessages:alsoTryRadicalRed"),
    i18next.t("splashMessages:eeveeExpo"),
    i18next.t("splashMessages:ynoproject"),
    i18next.t("splashMessages:breedersInSpace"),
  ]);

  return [    i18next.t("splashMessages:breedersInSpace"),];
}

```

reload a couple of times and it should show up eventually
## Checklist
- [X] There is no overlap with another PR?
- [X] The PR is self-contained and cannot be split into smaller PRs?
- [X] Have I provided a clear explanation of the changes?
- [X] Have I tested the changes (manually)?
    - [X] Are all unit tests still passing? (`npm run test`)
- [X] Are the changes visual?
  - [X] Have I provided screenshots/videos of the changes?

### CODE REVIEW

**Review by CodeTappert — APPROVED** [review]

File: src/locales/pt_BR/splash-messages.ts [hunk 1]
```diff
@@ -34,4 +34,5 @@ export const splashMessages: SimpleTranslationEntries = {
   "alsoTryRadicalRed": "Também Jogue Radical Red!",
   "eeveeExpo": "Eevee Expo!",
   "ynoproject": "YNOproject!",
+  "spaceBreeders": "Space Breeders!",
```
Comment (josericardo-fo) [inline src/locales/pt_BR/splash-messages.ts:37]: Criadores Pokémon no Espaço!

Comment (josericardo-fo) [inline src/locales/pt_BR/splash-messages.ts:37]:
```suggestion
  "spaceBreeders": "Criadores Pokémon no Espaço!!",
```

**Comment by torranx** [comment]
if this is in reference to breeders in space biome, isnt it more appropriate to word it like "Breeders In Space?!". The current text sounds like the person is breeding space to me.

**Comment by flx-sta** [comment]
@torranx yeah.. I was a bit torn between those two wordings.. ironically the German translation is designed in that way of a sentence

**Comment by flx-sta** [comment]
@josericardo-fo can you double check if the translation of pt_br is still accurate with the new wording?

**Comment by flx-sta** [comment]
@Adri1 french?

File: src/locales/fr/splash-messages.ts [hunk 2]
```diff
@@ -34,4 +34,5 @@ export const splashMessages: SimpleTranslationEntries = {
   "alsoTryRadicalRed": "Essaye aussi Radical Red !",
   "eeveeExpo": "Eevee Expo !",
   "ynoproject": "YNOproject !",
+  "breedersInSpace": "Breeders in space!",
```
Comment (Adri1) [inline src/locales/fr/splash-messages.ts:37]:
```suggestion
  "breedersInSpace": "Des Éleveurs dans l’espace !",
```

**Review by Adri1 — COMMENTED** [review]
Added French

**Review by Adri1 — APPROVED** [review]

**Comment by flx-sta** [comment]
@f-fsantos this would be ready to be merged

**Comment by flx-sta** [comment]
Waiting for some more translations

File: src/locales/pt_BR/splash-messages.ts [hunk 3]
```diff
@@ -34,4 +34,5 @@ export const splashMessages: SimpleTranslationEntries = {
   "alsoTryRadicalRed": "Também Jogue Radical Red!",
   "eeveeExpo": "Eevee Expo!",
   "ynoproject": "YNOproject!",
+  "breedersInSpace": "Criadores Pokémon no Espaço!!",
```
Comment (josericardo-fo) [inline src/locales/pt_BR/splash-messages.ts:37]:
```suggestion
  "breedersInSpace": "Criadores Pokémon no Espaço!",
```

File: src/locales/es/splash-messages.ts [hunk 4]
```diff
@@ -34,4 +34,5 @@ export const splashMessages: SimpleTranslationEntries = {
   "alsoTryRadicalRed": "¡Prueba también Radical Red!",
   "eeveeExpo": "¡Eevee Expo!",
   "ynoproject": "¡YNOproject!",
+  "breedersInSpace": "Breeders in space!",
```
Comment (AsdarDevelops) [inline src/locales/es/splash-messages.ts:37]:
```suggestion
  "breedersInSpace": "Criadores... ¡EN EL ESPACIO!",
```

**Review by AsdarDevelops — CHANGES_REQUESTED** [review]
Spanish localization

**Comment by NicusPulcis** [comment]
@flx-sta added italian

File: src/locales/zh_CN/splash-messages.ts [hunk 5]
```diff
@@ -34,4 +34,5 @@ export const splashMessages: SimpleTranslationEntries = {
   "alsoTryRadicalRed": "也玩玩看激进红！",
   "eeveeExpo": "伊布博览会！",
   "ynoproject": "Yume Nikki 页游项目！",
+  "breedersInSpace": "Breeders in space!",
```
Comment (40chyan) [inline src/locales/zh_CN/splash-messages.ts:37]:
```suggestion
  "breedersInSpace": "饲养员也能上太空？",
```

**Review by 40chyan — COMMENTED** [review]
cn

**Comment by flx-sta** [comment]
@Tempo-anon @f-fsantos reasy to merge. (Just 2 languages missing)

**Comment by NicusPulcis** [comment]
@flx-sta i've sent a PR for the italian translation

**Comment by flx-sta** [comment]
> @flx-sta i've sent a PR for the italian translation

thanks. A simple `suggestion` in the file works too next time. Takes less time to create

**Comment by NicusPulcis** [comment]
aaaaaalright @flx-sta thanks for the tip! i'll try to remember how that works for the next time lol

**Review by CodeTappert — APPROVED** [review]

**Review by Madmadness65 — APPROVED** [review]

**Pull request merged**